medusa: add page (#3899)
tldr-pages/tldr: pages/linux/medusa.md

# Medusa

> > A speedy, parallel, and modular, login brute-forcer.

Reviewer: We tend to omit adjectives ("speedy", "top quality") in the main description. Stick to describing what it is and what it does. "Parallel" and "Modular" is ok here in my opinion because it only describes a feature of the tool.
I've left a suggestion below. 
```suggestion
> A modular and parallel login brute-forcer for a variety of protocols.
```

> - General:

Reviewer: Please use verbs and full sentences to describe the functionality here.
I've left a suggestion but it probably does not fully describe what the command does so I would ask you to take that into consideration and rephrase it.
Same for the next examples. 
```suggestion
- Execute brute force using a list of usernames and passwords:
```

> # Medusa

Reviewer 2: Medusa should be lower case.

> A modular and parallel login brute-forcer for a variety of protocols.

> - Execute brute force against a ftp server using a file containing usernames and a file containing passwords:

Reviewer: Just a minor suggestion:
```suggestion
- Execute brute force against an FTP server using a file containing usernames and a file containing passwords:
```

> `medusa -M ftp -h host -U {{username_file}} -P {{password_file}}`

Reviewer 3: ```suggestion
`medusa -M ftp -h host -U {{path/to/username_file}} -P {{path/to/password_file}}`
```
Keep consistent with other pages.

> - Execute a login attempt against a HTTP server using the username, password and user-agent specified:

Reviewer 2: an HTTP server

`medusa -M HTTP -h host -u {{username}} -p {{password}} -m USER-AGENT:"{{Agent}}"`

- Execute a brute force against a MySQL server using a file cointaining usernames and a hash:

> `medusa -M mysql -h host -U {{username_file}} -p {{hash}} -m PASS:HASH`

Reviewer 3: ```suggestion
`medusa -M mysql -h host -U {{path/to/username_file}} -p {{hash}} -m PASS:HASH`
```

- Execute a brute force against a list of SMB server using a username and a pwdump file:

> `medusa -M smbnt -H {{hosts_file}} -C {{pwdump_file}} -u {{user}} -m PASS:HASH`

Reviewer 3: ```suggestion
`medusa -M smbnt -H {{hosts_file}} -C {{path/to/pwdump_file}} -u {{user}} -m PASS:HASH`
```
